docs/source/roles/system_manager/manage_deployments.md

```diff
@@ -26,6 +26,22 @@ PS> ./Update_SRE_SSL_Certificate.ps1 -shmId <SHM ID> -sreId <SRE ID>
 - where `<SHM ID>` is the {ref}`management environment ID <roles_deployer_shm_id>` for this SHM
 - where `<SRE ID>` is the {ref}`secure research environment ID <roles_deployer_sre_id>` for this SRE
 
+(renew_sas)=
+
+## {{locked_with_key}} Renew SRE Container Access Policies
+
+The [SRE storage containers](role_researcher_user_guide_shared_storage) for input data, backup and output are all provided by blob storage.
+The SRDs use [SAS tokens](https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview) bound to a [Stored Access Policy](https://learn.microsoft.com/en-us/azure/storage/common/storage-stored-access-policy-define-dotnet) to authenticate and access the data.
+
+When the containers are deployed the Stored Access Policy is valid for one year.
```

Comment:
Not true as currently written.

Reply:
The "valid for one year" part?

I believe it is because the default value is overwritten with a hard-coded value of 1

https://github.com/alan-turing-institute/data-safe-haven/blob/e8d67c7fae8cd23e265132f10792d5cd826a26f9/deployment/secure_research_environment/setup/Setup_SRE_Storage_Accounts.ps1#L64-L68